server/database/data/items.json

```diff
@@ -0,0 +1,222 @@
+[
+  {
+    "name": "Sinistrofu Cloak",
+    "item_type": "Cloak",
+    "set": "Sinistrofu Set",
+    "level": "200",
+    "stats": [
+      {
+        "stat": "Vitality",
+        "description": "251 to 300 Vitality",
+        "maxStat": "300"
```

Comment:
What's the reasoning behind structuring stats like this? Would it be better to store the min stat as well, and then generate the "description" string? Also, shouldn't maxStat be a number?

Reply:
I figured that, in practice, we only ever use the max string and then the full description. If we only ever need those two things, I thought it would be better to provide them directly.

And yea, I'll modify it to be an Int

Reply:
My concern is that you're duplicating the information - if we need to change something, we will need to change it in two places and that could cause problems later